src/dev/arm/SMMUv3.py

```diff
@@ -74,10 +74,11 @@ class SMMUv3(ClockedObject):
     type = 'SMMUv3'
     cxx_header = 'dev/arm/smmu_v3.hh'
 
-    master = MasterPort('Master port')
-    master_walker = MasterPort(
+    master = RequestPort('Master port')
+    master_walker = RequestPort(
         'Master port for SMMU initiated HWTW requests (optional)')
-    control = SlavePort('Control port for accessing memory-mapped registers')
+    control =
```

Comment:
This is a bit of a strange way to solve the line length problem. I suggest doing it like the line above and putting `ResponsePort(` on this line and the "help" on the next line.